packages/mobile/src/paymentRequest/utils.ts

```diff
@@ -49,7 +49,7 @@ export function getSenderFromPaymentRequest(
   } else {
     return {
       kind: RecipientKind.MobileNumber,
-      address: paymentRequest.requesterAddress,
+      address: paymentRequest.requesteeAddress,
```

Comment:
This seems like it was broken before maybe?

Reply:
Yes!